ubuntu/README.md

```diff
@@ -0,0 +1,673 @@
+  [185]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u131-8.21.0.1/Dockerfile
+  [186]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u144-8.23.0.3/Dockerfile
+  [187]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u152-8.25.0.1/Dockerfile
+  [188]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u162-8.27.0.7/Dockerfile
+  [189]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u172-8.30.0.1/Dockerfile
+  [190]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u181-8.31.0.1/Dockerfile
+  [191]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u192-8.33.0.1/Dockerfile
+  [192]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u202-8.36.0.1/Dockerfile
+  [193]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u212-8.38.0.13/Dockerfile
+  [194]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u222-8.40.0.25/Dockerfile
+  [195]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u232-8.42.0.21/Dockerfile
+  [196]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u232-8.42.0.23/Dockerfile
+  [197]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u242-8.44.0.11/Dockerfile
+  [198]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u252-8.46.0.19/Dockerfile
+  [199]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u262-8.48.0.51/Dockerfile
+  [200]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u272-8.50.0.21/Dockerfile
+  [201]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u275-8.50.0.53/Dockerfile
+  [202]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u282-8.52.0.23/Dockerfile
+  [203]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u302-8.56.0.21/Dockerfile
+  [204]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u312-8.58.0.13/Dockerfile
+  [205]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u322-8.60.0.21/Dockerfile
+  [206]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u332-8.62.0.19/Dockerfile
+  [207]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u342-8.64.0.15/Dockerfile
+  [208]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u345-8.64.0.19/Dockerfile
+  [209]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u352-8.66.0.15/Dockerfile
+  [210]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u362-8.68.0.19/Dockerfile
+  [211]: https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u362-8.68.0.21/Dockerfile
```

Comment:
File not found, https://github.com/it-mak/zulu-openjdk/tree/feature/move-ubuntu/ubuntu/8u362-8.68.0.21/ -- docker files had platform suffix at that moment (e.g. Dockerfile_x86)

Reply:
All Dockerfile_x86 Dockerfile_arm were combined to one Dockerfile as it was at the start.